src/components/TagsField.js

```diff
@@ -0,0 +1,70 @@
+/* @flow */
+import React, { Component } from "react";
+
+
+function toTagList(tagsString: string, unique: bool = false): string[] {
```

Comment:
Can we add an unittest for this specific `toTagList` fucntion? I have the feeling that we will need to handle specific edge case with time.